src/dev/impl/DevToys/Helpers/TimestampToolHelper.cs

```diff
@@ -0,0 +1,108 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using DevToys.Core;
+using DevToys.Models;
```

Comment:
It looks like several of these `using` statements are useless. Pro tip: in Visual Studio, you can use `Ctrl+R, Ctrl+G` to re-order and clear the `using` statements automatically :)